templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -23,7 +23,7 @@
         data-qa="page-nav-toggle"
         @click="opened = !opened"
       >
-        <i v-if="!opened" class="fas fa-bars fa-lg"></i>
+        <i v-if="!opened" class="fas fa-bars fa-lg" style="color: black;"></i>

```

Comment:
Ideally, the color should be applied to the element that has a background color, not on the individual children. We want to reduce the number of CSS rules, so it's better to apply it as far up the dom chain as possible to reduce the CSS rules. Remember that if a parent element has a certain color, the child will inherit it unless there's an exception somewhere. If there is an exception, we should remove that exception, preferably. 

If there are WP css, we should override those in the tapestry.css file instead rather than individual components.

Reply:
This is the for the navigation sidebar, where every button by default inherits a `color: #fff`. Since the background is changed to a light colour, it's necessary to override the colour of the button to be dark here.
![image](https://user-images.githubusercontent.com/11191061/129641769-598ceeeb-bc31-4cea-88b7-adf98d2c9f25.png)